frontend/src/components/DataProductLink.jsx

```diff
@@ -1,16 +1,17 @@
-import React from "react"
-import settings from "../settings"
+import React, { useContext } from "react"
+import DataspaceConfigurationContext from "../context/dataspaceConfigurationContext"
 
 export default function DataProductLink({ definition }) {
-  const link = `${settings.DATA_DEFINITION_VIEWER_URL}/definitions/${definition}`
+  const dataSpaceConfiguration = useContext(DataspaceConfigurationContext)
+
   return (
     <span>
-      <a href={link} target="_blank">
+      <a href={dataSpaceConfiguration.definitionViewerUrl} target="_blank">
         {definition}
       </a>{" "}
       data product published under <strong>ioxio</strong> source on{" "}
-      <a href={settings.IOXIO_SANDBOX_DATASPACE_URL} target="_blank">
-        IOXIO Sandbox
+      <a href={dataSpaceConfiguration.dataspaceBaseUrl} target="_blank">
+        {dataSpaceConfiguration.dataspaceName.trim()}
```

Comment:
Would you mind changing this so that the whole official name of the dataspace is used as the link; i.e. no splitting off of the `Dataspace` and trimming.